src/components/Quiz/QuizzesStats.tsx

```diff
@@ -0,0 +1,193 @@
+    `https://twitter.com/intent/tweet?text=${tweet}&hashtags=${hashtags}`
+  )
+}
+
+const handleShare = ({ score, total }: QuizShareStats) =>
+  shareOnTwitter({
+    score,
+    total,
+  })
+
+const QuizzesStats: React.FC = () => {
+  const { score: userScore, completed, average } = useContext(QuizzesHubContext)
+  const totalQuizzesNumber =
+    ethereumBasicsQuizzes.length + usingEthereumQuizzes.length
+  const TOTAL_QUIZZES_POINTS = getTotalQuizzesPoints()
+  const numberOfCompletedQuizzes = getNumberOfCompletedQuizzes(
+    JSON.parse(completed)
+  )
+
+  return (
+    <Box flex={1} order={{ base: 1, lg: 2 }} w="full">
+      <Stack mt={{ base: 0, lg: 12 }} gap={{ base: 8, lg: 4 }}>
+        {/* user stats */}
+        <Grid
+          gap={4}
+          bg="ednBackground"
+          borderRadius={{ base: "none", lg: "lg" }}
+          border="none"
+          p={{ base: 8, lg: 12 }}
+          mb={-2}
+        >
+          <GridItem colSpan={{ base: 2, lg: 1 }} alignSelf="center" order={1}>
+            <Text
+              color="body"
+              fontWeight="bold"
+              fontSize="xl"
+              margin={0}
+              textAlign={{ base: "center", lg: "left" }}
+            >
+              <Translation id="your-total" />
+            </Text>
+          </GridItem>
+
+          <GridItem
+            colSpan={{ base: 2, lg: 1 }}
+            justifySelf={{ base: "auto", lg: "end" }}
+            alignSelf="center"
+            order={{ base: 3, lg: 2 }}
+          >
+            <Button
+              variant="outline-color"
+              leftIcon={<Icon as={FaTwitter} />}
+              onClick={() =>
+                handleShare({ score: userScore, total: TOTAL_QUIZZES_POINTS })
+              }
+              w={{ base: "full", lg: "auto" }}
+              mt={{ base: 2, lg: 0 }}
+            >
+              <Translation id="share-results" />
+            </Button>
+          </GridItem>
+
+          <GridItem colSpan={2} order={{ base: 2, lg: 3 }}>
+            <Stack gap={2}>
+              <Flex
+                justifyContent={{ base: "center", lg: "flex-start" }}
+                alignItems="center"
+              >
+                <Circle size="64px" bg="primary" mr={4}>
+                  <TrophyIcon color="neutral" w="35.62px" h="35.62px" />
+                </Circle>
+
+                <Text fontWeight="bold" fontSize="5xl" mb={0} color="body">
+                  {userScore}
+                  <Text as="span" color="bodyLight">
+                    /{TOTAL_QUIZZES_POINTS}
+                  </Text>
+                </Text>
+              </Flex>
+
+              <Progress value={userScore} />
+
+              <Flex direction={{ base: "column", lg: "row" }}>
+                <Text mr={10} mb={0} mt={{ base: 2, lg: 0 }} color="bodyLight">
+                  <Translation id="average-score" />{" "}
+                  <Text as="span" color="body">
+                    {average}%
+                  </Text>
```

Comment:
If the user, the designs show an 'N/A' state.

<img width="238" alt="Screenshot 2023-05-29 at 11 17 22" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/4558448b-f12f-4daf-907d-eb32826fe92b">


Not sure about this tbh because of translations. What do you think @konopkja?

Reply:
Spoke to @konopkja, this should just be 'Average score: --' (two dashes) if the average is 0.

Blocked by #10349

Reply:
@nhsz lets replace "NA" with "--" for empty state on the Average score

Reply:
@minimalsm @konopkja why not use `0%` as currently? percentage is a number

Reply:
@konopkja if you ahven't done any quiz/question yet, your user score is 0 and then your average score, makes more sense to me than using a different notation for the average

Reply:
Will keep the number for now